### DISCUSSION TITLE
Cook Discussions Here!

### DISCUSSION THREAD

**Discussion by glitchedgitz** [giteshnxtlvl/cook#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, you can create new discussions.

**Comment by acrawford73** [comment]
Hi, I am interested in your project. I am new to Go language. May you please help why I cannot run your script.
`ubuntu@u18d:~$ git clone https://github.com/giteshnxtlvl/cook.git
Cloning into 'cook'...
remote: Enumerating objects: 312, done.
remote: Counting objects: 100% (89/89), done.
remote: Compressing objects: 100% (46/46), done.
remote: Total 312 (delta 76), reused 43 (delta 43), pack-reused 223
Receiving objects: 100% (312/312), 2.48 MiB | 8.88 MiB/s, done.
Resolving deltas: 100% (154/154), done.
ubuntu@u18d:~$ cd cook
ubuntu@u18d:~/cook$ ll
total 40K
-rw-rw-r-- 1 ubuntu ubuntu 3.6K Apr 27 18:58 cook.yaml
drwxrwxr-x 2 ubuntu ubuntu 4.0K Apr 27 18:58 images
-rw-rw-r-- 1 ubuntu ubuntu 1.1K Apr 27 18:58 LICENSE
-rw-rw-r-- 1 ubuntu ubuntu 8.7K Apr 27 18:58 main.go
-rw-rw-r-- 1 ubuntu ubuntu 7.1K Apr 27 18:58 parseinput.go
-rw-rw-r-- 1 ubuntu ubuntu 7.4K Apr 27 18:58 README.md
ubuntu@u18d:~/cook$ pwd
/home/ubuntu/cook
ubuntu@u18d:~/cook$ export COOK=/home/ubuntu/cook
ubuntu@u18d:~/cook$ env | grep COOK
COOK=/home/ubuntu/cook
ubuntu@u18d:~/cook$ go run main.go -config
# command-line-arguments
./main.go:140:10: undefined: strings.ReplaceAll
./main.go:160:23: undefined: strings.ReplaceAll
./main.go:200:2: undefined: parseInput
./main.go:211:20: undefined: parseIntRanges
./main.go:217:41: undefined: parseValue
./main.go:246:19: undefined: columnCases
./main.go:249:20: undefined: columnCases
./main.go:259:21: undefined: columnCases
./main.go:267:21: undefined: columnCases
./main.go:275:21: undefined: columnCases
./main.go:275:21: too many errors`
